script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -86,7 +86,7 @@ Our wordless march continues, punctuated only by the sound of our footsteps.
 [offset:17255]{
 -- Page 279, Offset 17255.
 -- ―――と。
-―――Until.
+―――Then.
```

Comment:
Kinda prefer until here, since it preserves the overall sentence flow a bit better
> Our wordless march continues, punctuated only by the sound of our footsteps until Arcueid abruptly stops [...]